xml/debuginfod.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="debuginfo">
+ <title>Getting debugging data and sources with &debuginfod;</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="debuginfo-introduction">
+  <title>Introduction</title>
+
+  <para>
+   Developers often need to solve bugs in software packages. Debugging tools,
+   for example GDB, use information contained in special files that are not
```

Comment:
Can you please verify if `debuginfod`  is enabled in the GDB version we ship in `SLE15SP4`? If not, we should emphasize that as it is a primary use case.

Reply:
it is not :-(

Reply:
I'm going to ask my colleague if we can enable the support as SLE15SP4 contains the latest release of gdb.

Reply:
i got the info from Michael Matz that default support in gdb is not going to be enabled in SLE 15 product line

Reply:
However, there's a high chance we can deliver the functionality in the next Toolchain module update (Autumn 2022).